src/coreclr/tests/src/readytorun/crossgen2/Program.cs

```diff
@@ -1249,6 +1274,16 @@ private static bool DelegateFromAnotherModuleTest()
             return true;
     }
 
+    private static bool SealedDefaultInterfaceMethodTest()
+    {
+        IGenericWithSealedDefaultMethod<string> igsdf = new ImplementGenericWithSealedDefaultMethod();
```

Comment:
I like this test, but please add another variant where the default method interface is in helperdll.dll not in this dll.

Reply:
yup good call on adding cross module test. Added and switched the interface check to be first.